src/frontend/komponenter/Fagsak/Vilkårsvurdering/VilkårsvurderingPeriode/VilkårsvurderingPeriodeSkjema.tsx

```diff
@@ -236,8 +226,73 @@ const VilkårsvurderingPeriodeSkjema: React.FC<IProps> = ({
         skjema.visFeilmeldinger &&
         skjema.felter.vilkårsresultatvurdering.valideringsstatus === Valideringsstatus.Feil;
 
+    const handleNavigering = async (handling: PeriodeHandling) => {
```

Comment:
Ville også her foreslått et map for å hente ut riktig handling, og vurdere denne strukturen for å tydliggjøre valideringslogikken:

```
const handleNavigering = async (handling: PeriodeHandling) => {
  if (harUlagredeData) {
    if (!validerOgOppdaterFelter(periode)) return;
    settNavigerer(true);
    return await sendInnSkjemaOgNaviger(handling);
  }
  return navigeringsFunksjoner[handling]?.();
};

```